index.html

```diff
@@ -1,105 +1,120 @@
+    <link rel="stylesheet" href="css/theme/black.css">
 
-		<!-- Theme used for syntax highlighting of code -->
-		<link rel="stylesheet" href="lib/css/zenburn.css">
+    <!-- Theme used for syntax highlighting of code -->
+    <link rel="stylesheet" href="lib/css/zenburn.css">
 
-		<!-- Printing and PDF exports -->
-		<script>
-			var link = document.createElement( 'link' );
-			link.rel = 'stylesheet';
-			link.type = 'text/css';
-			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
-			document.getElementsByTagName( 'head' )[0].appendChild( link );
-		</script>
-	</head>
-	<body>
-		<div class="reveal">
-			<div class="slides">
-				<section>
-					<h1>{Logo Goes Here}</h1>
-					<h2>Blogging Evening</h2>
-					<h3>April 27th @ <a href="#">GoCardless</a></h3>
-					</section>
+    <!-- Printing and PDF exports -->
+    <script>
+      var link = document.createElement( 'link' );
+      link.rel = 'stylesheet';
+      link.type = 'text/css';
+      link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
+      document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section>
+          <h1>{Logo Goes Here}</h1>
+          <h2>Blogging Evening</h2>
+          <h3>April 27th @ <a href="#">GoCardless</a></h3>
+          </section>
 
-				<section>
-					<h1>Why Blog?</h1>
-				</section>
+        <section>
+          <h1>Why Blog?</h1>
+        </section>
 
-				<section>
-					<h1>Tips from <a href="#">Ben Nadel</a></h1>
-					<ol>
-						<li>Keep a backlog of ideas that you want to explore</li>
-						<li>Blog for yourself</li>
-						<li>Your point of view is always unique</li>
-						<li>Try never to make assertions that you haven't tested</li>
-						<li>No topic is too small or insignificant</li>
-					</ol>
-				</section>
+        <section>
+          <h1>Tips from <a href="#">Ben Nadel</a></h1>
+          <ol>
+            <li>Keep a backlog of ideas that you want to explore</li>
+            <li>Blog for yourself</li>
+            <li>Your point of view is always unique</li>
+            <li>Try never to make assertions that you haven't tested</li>
+            <li>No topic is too small or insignificant</li>
+          </ol>
+        </section>
 
-				<section>
-					<h1>Example Topics</h1>
-					<ul>
-						<li>What is SEO? And how do I apply it to my website?</li>
-						<li>Linters what are they? Why do I need to use them?</li>
-						<li>What is Object Oriented programming?</li>
-						<li>Heroku 101</li>
-						<li>How to set DNS server to point to amazon s3 bucket?</li>
-						<li>How to center anything with CSS</li>
-						<li>Which Javascript framework should i choose?</li>
-						<li>Mobile first design and development</li>
-						<li>Bootstrap vs Foundation?</li>
-					</ul>
-				</section>
+        <!-- Elizabeth's -->
```

Comment:
Comment isn't necessary. Your heading is `Elizabeth's Blogging` so it's pretty obvious it's your section 😈